AeroGearSdkExample/accesscontrol/AccessControlViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  AccessControlViewController.swift
+//  secure-ios-app
+//
+//  Created by Wei Li on 14/12/2017.
+//  Copyright © 2017 Wei Li. All rights reserved.
```

Comment:
Remove those comments as they are misleading. There is no copyright and license is apache.